spec/commands/feed/find_new_stories_spec.rb

```diff
@@ -1,62 +1,60 @@
 # frozen_string_literal: true
 
 RSpec.describe Feed::FindNewStories do
-  context "the feed contains no new stories" do
-    before { allow(StoryRepository).to receive(:exists?).and_return(true) }
+  def create_entry(**options)
+    entry = {
+      published: nil,
+      title: "story1",
+      url: "www.story1.com",
+      content: "",
+      id: "www.story1.com",
+      **options
+    }
+    double(entry)
+  end
+
+  def create_raw_feed(feed, entries: [])
+    xml = GenerateXml.call(feed, entries)
+    Feedjira.parse(xml)
+  end
 
+  context "the feed contains no new stories" do
     it "finds zero new stories" do
-      story1 = double(published: nil, id: "story1")
-      story2 = double(published: nil, id: "story2")
-      feed   = double(entries: [story1, story2])
-      time = Time.zone.local(2013, 1, 2)
-      result = described_class.call(feed, 1, time)
+      feed = create(:feed)
+      create(:story, entry_id: "www.story1.com", feed_id: feed.id)
+      raw_feed = create_raw_feed(feed, entries: [create_entry])
+      result = described_class.call(raw_feed, feed.id)
+
       expect(result).to be_empty
     end
   end
 
   context "the feed contains new stories" do
     it "returns stories that are not found in the database" do
-      story1 = double(published: nil, id: "story1")
-      story2 = double(published: nil, id: "story2")
-      feed   = double(entries: [story1, story2])
-
-      allow(StoryRepository)
-        .to receive(:exists?).with("story1", 1).and_return(true)
-      allow(StoryRepository)
-        .to receive(:exists?).with("story2", 1).and_return(false)
-
-      time = Time.zone.local(2013, 1, 2)
-      result = described_class.call(feed, 1, time)
-      expect(result).to eq([story2])
+      feed = create(:feed)
+      create(:story, entry_id: "www.story1.com", feed_id: feed.id)
+      entry = create_entry(published: Time.zone.now, url: "www.story2.com")
+      raw_feed = create_raw_feed(feed, entries: [entry])
+
+      expect(described_class.call(raw_feed, feed.id)).to eql(raw_feed.entries)
```

Comment:
It's not a huge deal since they are equivalent in this case, but I generally us `eq` instead of `eql`. The [differences are subtle](https://stackoverflow.com/a/54088700/372479) and in many cases irrelevant, but for the sake of consistency I think it's probably worth sticking with `eq`.

```suggestion
      expect(described_class.call(raw_feed, feed.id)).to eq(raw_feed.entries)
```